src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+
+@Entity
 public class Actor implements Cloneable {
```

Comment:
```suggestion
@Entity
@Table(name = "actors")
public class Actor implements Cloneable {
```
It would be better to add table name